lib/AST/NameLookupRequests.cpp

```diff
@@ -545,8 +545,10 @@ static UnqualifiedLookupDescriptor excludeMacrosIfNeeded(
           UnqualifiedLookupFlags::ExcludeMacroExpansions))
     return descriptor;
 
-  auto &evaluator = descriptor.DC->getASTContext().evaluator;
-  if (!evaluator.hasActiveResolveMacroRequest())
```

Comment:
@DougGregor is there any reason to keep `hasActiveResolveMacroRequest()` around? It's now unused.

Reply:
No, let's get rid of it! Your approach is far better, and I don't want us accidentally thinking we should use that operation.